util/find_missing_timeline_translations.js

```diff
@@ -88,7 +88,7 @@ function findMissingRegex() {
         let replace = Regexes.parse(regex);
         if (transRegex.match(replace))
           foundMatch = true;
-        transRegex = transRegex.replace(replace, set[regex]);
+        transRegex = transRegex.split(replace).join(set[regex]);
```

Comment:
Can you explain to me what this part is doing, sorry?

Reply:
This is an equivalent of .replaceAll() in other languages

The old version only replaced the first occurence and not all (e.g. shinryus wings were not correctly handeled)